docker-compose.demo.yml

```diff
@@ -80,9 +80,12 @@ services:
             - ./conductor/server/entrypoint:/srv
             - ./conductor/server/workflows:/srv/workflows
             - ./conductor/server/tasks:/srv/tasks
+            - ./conductor/server/events:/srv/events
         entrypoint: /srv/start_conductor.sh
         environment:
             - CONFIG_PROP=config.properties
+        ports:
+            - "8080:8080"
```

Comment:
@GregorioDiStefano will this mess around with tests ? I find this very useful when testing

Reply:
on osx docker you cannot access containers directly, they need to be exposed (differs from linux docker)

Reply:
https://github.com/mendersoftware/integration/blob/master/tests/run.sh#L21

Update this line to remove the 8080 port from being exposed while testing

Reply:
cool, done